introduction/07-faq.asciidoc

```diff
@@ -0,0 +1,59 @@
+== FAQ
+
+To conclude the Introduction segment of the Learning Path, here are some Frequently Asked Questions people have when embarking on their InnerSource journey.
+
+=== What is the cost/overhead of maintaining an InnerSource projects?
+It depends! An InnerSource project that encourages small pull requests and has clear contribution guidelines may require very little overhead, with most of the work being code reviews. To learn more about practices that can reduce the overheard of maintaining InnerSource projects, we suggest you look at the https://patterns.innersourcecommons.org/[InnerSource Patterns], especially:
+
+* https://patterns.innersourcecommons.org/p/30-day-warranty[30 Day Warranty]
+* https://patterns.innersourcecommons.org/p/service-vs-library[Service vs. Library]
+* https://patterns.innersourcecommons.org/p/base-documentation[Standard Base Documentation]
+* https://patterns.innersourcecommons.org/p/trusted-committer[Trusted Committer]
+
+=== What is the cost/overhead of contributing to an InnerSource projects?
+50% more effort to commit. 100% less effort to maintain.
+
+=== Why not just open source?
+By all means do so if the project makes sense! Some projects are specific to your company or are a competitive advantage, so you'll want to keep those as InnerSource. Some need to iterate more quickly than can be done in the open.
+
+If your organisation isn't familiar with running open source projects, InnerSource can help people learn the skills required with a view to open sourcing in the future.
+
+=== Will this slow us down?
+It depends on how far you're going. You'll probably be going a lot further than you think.
+
+image::https://user-images.githubusercontent.com/9609562/151901209-52b3468b-dedd-4319-9ca3-38b6b2bcfaf5.png[If you want to go fast, go alone. If you want to go far, go together]
+
+=== Will we just be reviewing PRs all day every day?
+If so then your core team is understaffed. A healthy team is staffed so that there is time for assisting contributors and making core contributions yourself.
+
+You can mitigate this by setting expectation, potentialy via SLAs. If contributors expect PR reviews within an hour, maybe you will be stuck reviewing PRs all the time, but if you set an SLA of 1 day or 1 week, this won't be the case.
+
+=== How do we convince management this is a good idea?
```

Comment:
From Chan:

Two recommendations:

Add link: [https://innersourcecommons.org/stories/](https://nam11.safelinks.protection.outlook.com/?url=https%3A%2F%2Finnersourcecommons.org%2Fstories%2F&data=05%7C01%7CRussell.Rutledge%40WellSky.com%7C56af9e6f875545d9803708da68eecf64%7C15d19784ad584a57a66fad1c0f826a45%7C0%7C0%7C637937672137212065%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=LmwIU%2BR27CAyhGzmD%2FMkUTok4HXTL7CINectJtukQFE%3D&reserved=0)
If your organization's OSPO manages innersource projects, reach out to them for support.

Reply:
[05940ed](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/465/commits/05940edfcdbc093fa7c2d92b956bf20b45a7ade2)